src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPageFairRecoveryOasis.java

```diff
@@ -57,16 +77,18 @@ public void adsRecoveryNoAdblockPageFairOasis() {
   )
   @Execute(asUser = User.USER_2)
   public void adsRecoveryLoggedInPageFairOasis() {
-    String url = urlBuilder.getUrlForPath(WIKIA, getUrlArticlePageFairRecovery());
+    String url = urlBuilder.getUrlForPath(WIKIA, getUrlArticlePageFairRecovery(true));
     AdsRecoveryObject adsRecoveryObject = new AdsRecoveryObject(driver, url, DESKTOP_SIZE);
 
     adsRecoveryObject.verifyNumberOfPageFairRecoveredSlots(0);
   }
 
-  private String getUrlArticlePageFairRecovery() {
+  private String getUrlArticlePageFairRecovery(boolean isPAL) {
```

Comment:
Did you consider not using acronyms for variable names? It's much easier to understand for a new person what's going on.